src/views/Gallery.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <div class="gallery-container">
+    <div class="gallery autoplay items-3">
+      <div id="item-1" class="control-operator"></div>
+      <div id="item-2" class="control-operator"></div>
+      <div id="item-3" class="control-operator"></div>
+
+      <figure class="item">
+        <img src="https://www.w3schools.com/bootstrap/chicago.jpg" />
+      </figure>
+
+      <figure class="item">
+        <img src="https://www.w3schools.com/bootstrap/la.jpg" />
+      </figure>
+
+      <figure class="item">
+        <img src="https://www.w3schools.com/bootstrap/ny.jpg" />
+      </figure>
+
+      <div class="controls">
+        <a href="#item-1" class="control-button">•</a>
+        <a href="#item-2" class="control-button">•</a>
+        <a href="#item-3" class="control-button">•</a>
+      </div>
+    </div>
+
+    <div>
+      <div class="grid-gallery">
```

Comment:
Can we separate this out into its own component and use `v-for` to make things cleaner?